onionperf/util.py

```diff
@@ -129,10 +129,14 @@ def find_ip_address_local():
     return ip_address
  
 def get_ip_address():
-    data = urllib.urlopen('https://check.torproject.org/').read()
-    ip_address = find_ip_address_url(data) 
+    ip_address = None
+    try:
+        data = urllib.urlopen('https://check.torproject.org/').read()
+        ip_address = find_ip_address_url(data)
+    except:
+        pass
     if ip_address is None:
```

Comment:
If it is at most once per measurement then we should be adding a warning to the logs every time a DNS failure occurs. We should not hide errors.